### ISSUE TITLE
[Bug] The cursor does not appear correctly

### ISSUE THREAD

**Issue by Huy-Ngo** [DevinR528/forget#1]
This in fact happened in the demo gif:

![The cursor appears at the bottom while the top item is selected](https://user-images.githubusercontent.com/19296926/107150711-ca4ab480-6991-11eb-95a2-2404c17604e3.png)

My example:

![The cursor appears on the incorrect row](https://user-images.githubusercontent.com/19296926/107150840-65dc2500-6992-11eb-8bbf-2c2de3541d8b.png)

The cursor apparently never appears in the first row. Notice that there appears to be an invisible cursor overlaying on the number `2` in my example.

Reproduction:

- Create a new sticky note. The cursor appears at the last position of the previous list.
- Create multiple to-do items. The cursor does not appears on the first item.
- Move down: The cursor moves with the selector.
- Move up: The cursor is below the selector.
- Move back to the previous note: the cursor sticks with the last item even though the selector goes back to the first item.

**Comment by Huy-Ngo** [comment]
Addition: deleting items makes the cursor sticks too.
![image](https://user-images.githubusercontent.com/19296926/107151248-231b4c80-6994-11eb-9b71-c3629ec6d5a8.png)

**Comment by DevinR528** [comment]
Would removing it all together be an acceptable solution? I don't remember how easily controlling the cursor would be using `termion` (the backend `tui` uses to control the terminal) but I imagine it would be do-able. Again if you are interested I would love a PR otherwise I can take a look in the next few weeks!

**Comment by Huy-Ngo** [comment]
Yup I think it's ok to remove the cursor, since the selector is already there.
